doc/api/process.md

```diff
@@ -472,8 +472,8 @@ added: 6.4.0
 
 * {string}
 
-The `process.argv0` property stores a read-only copy of the original value of
-`argv[0]` passed when Node.js starts.
+`process.argv0` is a configurable property that stores a read-only copy of the
+original value of `argv[0]` passed when Node.js starts.
```

Comment:
Considering that using this "feature" requires essentially an understanding of why you can't just assign to it, I'm leaning more heavily towards omitting the docs altogether (I'm sure there's plenty of properties in Node that aren't _intentionally_ configurable/writable but they're still there, and that fact isn't documented).

I think actually _using_ this feature is ideally something you only do when you have a very very specific idea of what you want to use it for, and it should be reeeeally rare.